ci/jenkins/test.sh

```diff
@@ -21,6 +21,7 @@ function echoerr {
 }
 
 KIND_CLUSTER=""
+JENKINS_LOCATION=""
```

Comment:
Should we set the default one as `lab`? otherwise, this script might do nothing when this condition `if [[ $IP_MODE == "ipv6" && "$TESTCASE" =~ "conformance" && $TESTBED_TYPE == "kind" ]]; then` matched with an empty JENKINS_LOCATION

@edwardbadboy @XinShuYang any thoughts for this?

Reply:
for now JENKINS_LOCATION=""  variable only for kind  testbed in this script so if tests are running on kind testbed then only we need check for Jenkins locations.